Library/Formula/rexster.rb

```diff
@@ -0,0 +1,15 @@
+require 'formula'
+
+class Rexster < Formula
+  homepage 'http://rexster.tinkerpop.com/'
+  url 'http://tinkerpop.com/downloads/rexster/rexster-server-2.6.0.zip'
+  sha1 '756e317dc4db1b1627b250f3442405518bd9fc6b'
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+
+    prefix.install_metafiles
+    prefix.install Dir["*"]
+  end
+
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!